docs/conf/cfp/east-2020.rst

```diff
@@ -0,0 +1,40 @@
+Write the Docs East 2020 Call for Venues
+========================================
+
+We're also excited to start planning another US conference in 2020.
+The goal will be to make closer to folks on the east coast,
```

Comment:
Perhaps "The goal is to make it closer to..." instead? I think "it" is missing after "make"